src/audio_spatial.c

```diff
@@ -11,6 +11,8 @@
 #include "tracks.h"
 #include "math_util.h"
 
+#define ABS(x) ((x) >= 0 ? (x) : -(x))
```

Comment:
Can you move this to include/macros.h? We seem to have some ABS macros in there although we also have something in include/sys/asm.h. Not sure if there is a potential for conflicts or not.
```C
#define ABS(x, y)   \
    .globl  x      ;\
    x   =   y
```

Reply:
Moved this to macros.h, and thankfully, it doesn't break anything.